pkg/flags/flags.go

```diff
@@ -75,6 +76,10 @@ var (
 		Name:  flagParallelTestsName,
 		Usage: "Run test features in parallel",
 	}
+	failFastFlag = flag.Flag{
+		Name:  flagFailFast,
+		Usage: "Fail the feature and rest of the features of the test on any test failure",
```

Comment:
I think this may need to be reworded.
`Fail immediately and stop running untested code` or something similar.

Reply:
Done.